src/qt/clientmodel.cpp

```diff
@@ -137,7 +137,7 @@ size_t ClientModel::getMempoolDynamicUsage() const
 
 double ClientModel::getVerificationProgress(const CBlockIndex* tipIn) const
 {
-    const CBlockIndex* tip = const_cast<const CBlockIndex* >(tipIn);
+    const CBlockIndex* tip = tipIn;
```

Comment:
Commit "Remove now-useless const_cast in qt/clientmodel"

Could remove this line and rename argument to `tip`.